import_export/resources.py

```diff
@@ -110,20 +110,41 @@ class ResourceOptions:
 
     skip_unchanged = False
     """
-    Controls if the import should skip unchanged records. Default value is
-    False
+    Controls if the import should skip unchanged records.
+    If ``True``, then each existing instance is compared with the instance to be
+    imported, and if there are no changes detected, the row is recorded as skipped,
+    and no database update takes place.
+
+    The advantages of enabling this option are:
+
+    #. Avoids unnecessary database operations which can result in performance
+       improvements for large datasets.
+
+    #. Skipped records are recorded in each :class:`~import_export.results.RowResult`.
+
+    #. Skipped records are clearly visible in the
+       :ref:`import confirmation page<import-process>`.
+
+    For the default ``skip_unchanged`` logic to work, the following options must also
+    be ``False`` (which is the default):
+
+    #. :attr:`~import_export.resources.ResourceOptions.skip_diff`
+
+    #. :attr:`~import_export.resources.ResourceOptions.skip_html_diff`
```

Comment:
Actually, why would `skip_html_diff` be required?

Reply:
Good point.  I have corrected this.